Furball.Vixie.Backends.OpenGL.Shared/TextureGL.cs

```diff
@@ -83,7 +83,7 @@ public class TextureGL : Texture, IDisposable {
         /// </summary>
         public bool IsFramebufferTexture {
             get;
-            internal init;
+            internal set;
```

Comment:
any reason for this change here?

Reply:
init doesnt exist on netstandard, it complains

Reply:
right i forgot